gen3-integration-tests/gen3_ci/jenkins_jobs/revoke-arborist-policy.groovy

```diff
@@ -0,0 +1,50 @@
+/*
+    String parameter USERNAME
+        e.g. cdis.autotest@gmail.com
+    String parameter POLICY
+        e.g.
+*/
+pipeline {
+    agent {
+        node {
+            label 'gen3-ci-worker'
+        }
+    }
+    stages {
+        stage('Clean old workspace') {
+            steps {
+                cleanWs()
+            }
+        }
+        stage('Initial Setup') {
+            steps {
+                // cloud-automation
+                checkout([
+                  $class: 'GitSCM',
+                  branches: [[name: 'refs/heads/master']],
+                  doGenerateSubmoduleConfigurations: false,
+                  extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'cloud-automation']],
+                  submoduleCfg: [],
+                  userRemoteConfigs: [[credentialsId: 'PlanXCyborgUserJenkins', url: 'https://github.com/uc-cdis/cloud-automation.git']]
+                ])
+            }
+        }
+        stage('Revoke Arborist Policy') {
+            steps {
+                dir("revoke-arborist-policy"){
+                    script {
+                        sh '''#!/bin/bash +x
+                        set -e
+                        export GEN3_HOME=\$WORKSPACE/cloud-automation
+                        export KUBECTL_NAMESPACE=\${NAMESPACE}
+                        source \$GEN3_HOME/gen3/gen3setup.sh
+
+                        gen3 devterm --namespace \${NAMESPACE} curl -X DELETE arborist-service/user/\${USERNAME}/policy/\${POLICY}
```

Comment:
can we not use arborist api here - https://petstore.swagger.io/?url=https://raw.githubusercontent.com/uc-cdis/arborist/master/docs/openapi.yaml#/user/delete_user__username__policy__policyName_